README.install.md

```diff
@@ -63,6 +63,21 @@ Alternatively, install Nmstate system-wide:
 ```shell
 pip uninstall -y nmstate; pip install .
 ```
+### Install nmstate from tarball
```

Comment:
I am not sure that this information is really useful since it should be only used by package maintainers - but we are the package maintainers. If it should stay in, there should be at least a recommendation to use the distributed version/Copr or PyPi instead of this method.

Reply:
Sure, I will add a recommendation.